FWCore/MessageLogger/interface/ELseverityLevel.h

```diff
@@ -18,120 +18,120 @@
-    const std::string& getInputStr() const;  // example: "ERROR"
-    const std::string& getVarName() const;   // example: "ELerror"
-
-    // -----  Emitter:
-    //
-    friend std::ostream& operator<<(std::ostream& os, const ELseverityLevel& sev);
-
-  private:
-    // Data per ELseverityLevel object:
-    //
-    int myLevel;
-
-  };  // ELseverityLevel
-
-  // ----------------------------------------------------------------------
-  // Declare the globally available severity objects,
-  // one generator function and one proxy per non-default ELsev_:
-  // ----------------------------------------------------------------------
-
-  constexpr const ELseverityLevel ELzeroSeverity{ELseverityLevel::ELsev_zeroSeverity};
-
-  constexpr const ELseverityLevel ELdebug{ELseverityLevel::ELsev_success};
-
-  constexpr const ELseverityLevel ELinfo{ELseverityLevel::ELsev_info};
-
-  constexpr const ELseverityLevel ELfwkInfo{ELseverityLevel::ELsev_fwkInfo};
-
-  constexpr const ELseverityLevel ELwarning{ELseverityLevel::ELsev_warning};
-
-  constexpr const ELseverityLevel ELerror{ELseverityLevel::ELsev_error};
-
-  constexpr const ELseverityLevel ELunspecified{ELseverityLevel::ELsev_unspecified};
-
-  constexpr const ELseverityLevel ELsevere{ELseverityLevel::ELsev_severe};
-
-  constexpr const ELseverityLevel ELhighestSeverity{ELseverityLevel::ELsev_highestSeverity};
-
-  // ----------------------------------------------------------------------
-  // Comparators:
-  // ----------------------------------------------------------------------
-
-  constexpr inline bool operator==(ELseverityLevel const& e1, ELseverityLevel const& e2) noexcept {
-    return e1.cmp(e2) == 0;
-  }
-  constexpr inline bool operator!=(ELseverityLevel const& e1, ELseverityLevel const& e2) noexcept {
-    return e1.cmp(e2) != 0;
-  }
-  constexpr inline bool operator<(ELseverityLevel const& e1, ELseverityLevel const& e2) noexcept {
-    return e1.cmp(e2) < 0;
-  }
-  constexpr inline bool operator<=(ELseverityLevel const& e1, ELseverityLevel const& e2) noexcept {
-    return e1.cmp(e2) <= 0;
-  }
-  constexpr inline bool operator>(ELseverityLevel const& e1, ELseverityLevel const& e2) noexcept {
-    return e1.cmp(e2) > 0;
-  }
-  constexpr inline bool operator>=(ELseverityLevel const& e1, ELseverityLevel const& e2) noexcept {
-    return e1.cmp(e2) >= 0;
-  }
-
-  // ----------------------------------------------------------------------
-
+  namespace messagelogger {
+    // ----------------------------------------------------------------------
+    // ELseverityLevel:
+    // ----------------------------------------------------------------------
+
+    class ELseverityLevel {
+    public:
+      // ---  One ELseverityLevel is globally instantiated (see below)
+      // ---  for each of the following levels:
+      //
+      enum ELsev_ {
+        ELsev_noValueAssigned = 0  // default returned by map when not found
+        ,
```

Comment:
If the other `ELseverityLevel::ELsev_` will be reformatted, reformat this one as well
```suggestion
        ELsev_noValueAssigned = 0,  // default returned by map when not found
```

Reply:
done